.code-samples.meilisearch.yaml

```diff
@@ -17,6 +19,16 @@ get_one_document_1: |-
   client.index('movies').document(25684)
 get_documents_1: |-
   client.index('movies').documents(limit: 2)
+document_guide_add_document_primary_key: |-
+  client.index('movies').add_documents([
+    {
+      reference_number: 287947,
+      title: 'Shazam',
+      poster: 'https://image.tmdb.org/t/p/w1280/xnopI5Xtky18MPhK40cZAGAOVeV.jpg',
+      overview: 'A boy is given the ability to become an adult superhero in times of need with a single magic word.',
+      release_date: '2019-03-23'
+    }
+  ], 'reference_number')
```

Comment:
I think the primary key is a named parameter

```
// lib/meilisearch/index.rb
add_documents(documents, primary_key = nil)
```

Reply:
nope! a named parameter in ruby is defined like this:

```rb
def add_documents(docs, primary_key: nil)
end

# usage:
add_documents([...], primary_key: "blabla")
add_documents([...]) # yeah, this is optional, since I defined a `nil` value as the default value.
```

but what we have is:

```rb
def add_documents(docs, primary_key = nil)
end

# usage:
add_documents([...], "blabla")
add_documents([...]) # yeah, this is still optional, since I defined a `nil` value as the default value.
```

Check it out:
[spec/meilisearch/index/documents_spec.rb](https://github.com/meilisearch/meilisearch-ruby/blob/main/spec/meilisearch/index/documents_spec.rb#L559)

Reply:
thanks a lot for the information :)